src/AuthLDAP.php

```diff
@@ -3074,7 +3074,7 @@ public function connect()
      * @param boolean $use_bind             do we need to do an ldap_bind? (true by default)
      * @param bool    $silent_bind_errors   Indicates whether bind errors must be silented
      *
-     * @return resource|false|\LDAP\Connection link to the LDAP server : false if connection failed
+     * @return resource|false link to the LDAP server : false if connection failed
```

Comment:
`|\LDAP\Connection` is a correct return type on PHP >= 8.1. It should be preserved.
```suggestion
     * @return resource|false|\LDAP\Connection link to the LDAP server : false if connection failed
```

Reply:
In PHP 8.3, I get this error:
```
Method AuthLDAP::connectToServer() has invalid return type LDAP\Connection.
```

Reply:
You can ignore this error (locally or per config).

Reply:
Do you have the LDAP extension locally (and loaded in CLI context) ?

Reply:
Yes :

![image](https://github.com/user-attachments/assets/e3d7b71d-d7e4-440a-9227-efb3e70684fe)